<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>串口调试助手</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>显示格式：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="baudrateSelect_comboBox">
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="portSelect_comboBox"/>
       </item>
       <item row="3" column="1">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="utf8_rbtn">
            <property name="text">
             <string>utf-8</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="hex_rbtn">
            <property name="text">
             <string>16进制</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="gbk_rbtn">
            <property name="text">
             <string>GBK</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>波特率：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="close_btn">
         <property name="text">
          <string>关闭串口</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="start_btn">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>开启串口</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>串口号：</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(175, 255, 207);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
